lib/joi/joi.bundle.js

```diff
@@ -37,14 +37,34 @@ module.exports =
 /******/ 	// define getter function for harmony exports
 /******/ 	__webpack_require__.d = function(exports, name, getter) {
 /******/ 		if(!__webpack_require__.o(exports, name)) {
-/******/ 			Object.defineProperty(exports, name, {
```

Comment:
Generated according to the instructions in [`lib/joi/README.md`](https://github.com/Kashoo/synctos/blob/d74fd36c6b0926d7f5f9b0dd5f52e9995ae554c7/lib/joi/README.md)